e2e/specs/text_area.spec.ts

```diff
@@ -68,6 +68,20 @@ describe("st.text_area", () => {
     );
   });
 
+  it("sets value correctly on command-enter keypress", () => {
```

Comment:
Was there a decision made to add support for `command-enter`? Not seeing it in the issue

Reply:
I was thinking maybe the user tried to use command instead of control, it's a common ux issue

Reply:
Gotcha, they said `ctrl-enter` in their issue comments. I'd maybe verify with them that they're using the correct combination before adding an extra one. We also tell them which keys to use in a helper message at the bottom right of the text area.

Reply:
I agree that we should check, just in case they _were_ using the `ctrl` key. Which would mean something else needs to be fixed.

That said, it makes sense to add support for `cmd-enter` too (even without updating the text area) since Mac users are so used to pressing `cmd` instead of `ctrl`.

Another solution would be so listen to `ctrl-enter` on Windows and Linux and `cmd-enter` on Mac. And update the message to match. If easy, then let's do this. Otherwise, the approach from this PR is fine too.

Reply:
Done, now i'm checking the navigator.platform to show Command or Ctrl but in any case both would work